point/gemini_commands.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import ipaddress
 from curses.ascii import isgraph
+from enum import *
```

Comment:
`import *` is typically discouraged (and fwiw, I might backtrack on doing similar imports in __init__.py at some point)